src/visualCaptcha/Captcha.php

```diff
@@ -299,7 +299,8 @@ private function getMimeType( $filePath ) {
             }
 
             // Just figure out from a set of possibilities, if we didn't figure it out before
-            $extension = end( explode('.', $filePath) );
+            $fileProperties = explode('.', $filePath);
+            $extension = end($filePropertiesp);
```

Comment:
There's an extra `p` in the variable name here, can you fix that and update the PR?